src/base/zonefile_fmt.rs

```diff
@@ -145,6 +145,7 @@ impl<W: fmt::Write> FormatWriter for SimpleWriter<W> {
 /// A single line writer that puts tabs between ungrouped tokens
 struct TabbedWriter<W> {
     first: bool,
+    first_block: bool,
```

Comment:
Maybe `first_in_block` is a bit more clear?

Reply:
But it's not first in any block, it's the start of the first block.